app/views/admin/scim_clients/edit.html.erb

```diff
@@ -32,7 +32,7 @@ See COPYRIGHT and LICENSE files for more details.
 <%=
   render(Primer::OpenProject::PageHeader.new) do |header|
     header.with_title { @scim_client.name }
-    header.with_description { t("admin.scim_clients.form.description_html", docs_url: "#") } # TODO: Deep link into documentation
+    header.with_description { t("admin.scim_clients.form.description_html", docs_url: ::OpenProject::Static::Links[:sysadmin_docs][:scim][:href]) }
```

Comment:
It was more than one location that needed to be covered here.

One other is in `new.html.erb` and the last one in `app/forms/scim_clients/form.rb`.

Reply:
As far as I can tell `new.html.erb` is still missing.